x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformContext.java

```diff
@@ -43,7 +43,7 @@
         this.currentCheckpoint = new AtomicLong(currentCheckpoint);
         this.taskListener = taskListener;
         this.failureCount = new AtomicInteger(0);
-        this.shouldStopAtCheckpoint = shouldStopAtCheckpoint;
+        this.shouldStopAtCheckpoint = false;
```

Comment:
Should it stay like this or is it some debug leftover?

Reply:
this was a bug I found, `shouldStopAtCheckpoint` is not a parameter, so this was a dump no-op:`a=a`

However, I will move the assignment to the member declaration (although `false` is default).